lib/main_publiccloud.pm

```diff
@@ -207,6 +207,12 @@ sub load_publiccloud_tests {
             load_publiccloud_download_repos();
         } elsif (get_var('PUBLIC_CLOUD_QAM')) {
             load_maintenance_publiccloud_tests();
+        } elsif (get_var('PUBLIC_CLOUD_PCW_CLIENT')) {
+            loadtest 'boot/boot_to_desktop';
+            loadtest 'publiccloud/ansible_client';
+        } elsif (get_var('PUBLIC_CLOUD_PCW_TARGET')) {
+            loadtest 'microos/disk_boot';
```

Comment:
it won't work like this see line 201

Reply:
```perl
if (check_var('PUBLIC_CLOUD_PREPARE_TOOLS', 1)) {
        load_create_publiccloud_tools_image();
    }
    elsif (check_var('PUBLIC_CLOUD_TOOLS_CLI', 1)) {
        load_publiccloud_cli_tools();
    }
elsif (get_var('PUBLIC_CLOUD_PCW_TARGET')) {
            loadtest 'microos/disk_boot';
            loadtest 'publiccloud/ansible_target';
}
    else {
        loadtest 'boot/boot_to_desktop';
        if (get_var('PUBLIC_CLOUD_MIGRATION')) {
            my $args = OpenQA::Test::RunArgs->new();
            loadtest('publiccloud/upload_image', run_args => $args);
            loadtest('publiccloud/migration', run_args => $args);
        } elsif (check_var('PUBLIC_CLOUD_DOWNLOAD_TESTREPO', 1)) {
            load_publiccloud_download_repos();
        } elsif (get_var('PUBLIC_CLOUD_QAM')) {
            load_maintenance_publiccloud_tests();
        } elsif (get_var('PUBLIC_CLOUD_PCW_CLIENT')) {
             loadtest 'publiccloud/ansible_client';
}
```

Reply:
Yes, understood. In my wayt a previous loadtest is added